src/inverseRobot.js

```diff
@@ -8,6 +8,31 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inversedRobot = {};
+  const arrKeys = [];
+  let repeat = false;
+
+  for (const key in robot) {
+    arrKeys.push(robot[key]);
+  }
+
+  for (let i = 0; i < arrKeys.length; i++) {
+    for (let j = i + 1; j < arrKeys.length; j++) {
+      if (arrKeys[i] === arrKeys[j]) {
+        repeat = true;
+      }
+    }
+  }
+
+  if (repeat === false) {
+    for (const key in robot) {
+      inversedRobot[robot[key]] = key;
+    }
+
+    return (inversedRobot);
+  } else {
+    return (null);
+  }
```

Comment:
It's to complicated. You should use for of loop for robot, check if inversedRobot already has value as key - if so - return null other way - add new value